src/components/MultiHopTrade/components/TradeAmountInput.tsx

```diff
@@ -405,7 +405,7 @@ export const TradeAmountInput: React.FC<TradeAmountInputProps> = memo(
                 onMaxClick={handleOnMaxClick}
                 onClick={onPercentOptionClick ?? noop}
               />
-              {balance && assetId && !label && (
+              {balance && assetId && !label && !isAccountSelectionHidden && (
```

Comment:
It is - it was an ommission from https://github.com/shapeshift/web/pull/6398, it should've been added both in inline and stacked branches.
We have two different branches depending on the layout (inline/stacked) and this was not honored when using a `stacked` layout - which is actually the default for `<TradeAmountInput />` (stake isn't affected because it's using `layout='inline'`).
Failure to fix this won't hide account selection in RFOX unstaking as it should.

Reply:
Note: https://github.com/shapeshift/web/commit/47a22a20e2a6b6b868c78c372d03523ce28b6f5f brings back `stacked` alignment